packages/flutter/test/material/switch_list_tile_test.dart

```diff
@@ -1044,7 +1044,7 @@ void main() {
     );
   });
 
-  testWidgets('SwitchListTile respects thumbIcon - M3', (WidgetTester tester) async {
```

Comment:
I assume these changes are because Material 3 is the default? Considering some Switch tests are design-language specific and some are not, I wonder if removing the M3 labels makes the tests a little more difficult to grasp at first glance? Just a throwaway comment.

Reply:
During Material 3 migration, we found some use cases which weren't tested in M2. In such cases, we added M3 specific tests. Now that M3 is defaults, these tests also become default and no longer require `M3` or `Material3` tag.

We've been adding Material 2 variants of these tests as part of larger M2-M3 test updates to make sure we also cover M2. (We may usually remove M2 test version of this).  Looks like this test wasn't updated for M2, I'll add the M2 variant.

Invited @justinmc to the doc where we track these test classes. He can add you, there you can check out test classes we have updated and find some examples of them.

Reply:
cc: @QuncCccccc who initiated this whole project.